core/templates/services/html-length.service.spec.ts

```diff
@@ -122,6 +122,9 @@ describe('Html Length Service', () => {
 
     const result = htmlLengthService.computeHtmlLength(htmlString, 'word');
 
+    /* "Hi this seems too good to be true but what to do
```

Comment:
Would you please write this and other descriptions more clearly, so it's easier for others to read and understand? For example:

"Hi this seems too good to be true but what to do man" is a paragraph with 13 words. The math tag counts as 1 word. So, this gives a total of 13 + 1 = 14 words.

Ditto elsewhere.

Reply:
Done!